pgx-utils/src/sql_entity_graph/pgx_sql.rs

```diff
@@ -1017,6 +1020,21 @@ fn connect_externs(
             );
         }
 
+        // The hash function must be defined after the {typename}_eq function.
+        for (hash_item, &hash_index) in hashes {
+            tracing::debug!(
```

Comment:
Could we prune out this message? I think it's extraneous.

Reply:
Good catch, done!